src/rgw/rgw_op.cc

```diff
@@ -4993,12 +4993,13 @@ int RGWCopyObj::verify_permission()
                                                         RGW_PERM_WRITE)){
         return -EACCES;
       }
-    }
-  } else if (! dest_bucket_policy.verify_permission(this, *s->auth.identity, s->perm_mask,
+    } else if (! dest_bucket_policy.verify_permission(this, *s->auth.identity, s->perm_mask,
                                                     RGW_PERM_WRITE)) {
     return -EACCES;
   }
 
+  }
```

Comment:
thanks @NancySu05, this looks right to me. could you please fix up the indentation inside this block? (lines 4997-4999 need an extra two spaces)

Reply:
I have changed according to your suggestion, please review ,thanks!